components/autofill/sql/create_shared_schema.sql

```diff
@@ -62,6 +69,7 @@ CREATE TABLE IF NOT EXISTS credit_cards_data (
 
 CREATE TABLE IF NOT EXISTS credit_cards_mirror (
     guid                TEXT NOT NULL PRIMARY KEY CHECK(length(guid) != 0),
+    -- Note that the entire payload here is encrypted in the same way cc_number_enc is locally.
```

Comment:
I initially read "in the same way" as meaning "using the same crypto algorithms", which IIUC is not the case; might be worth clarifying here with "(but using sync15's encryption scheme)" or similar?

Reply:
No, it does mean "using the same crypto as cc_number_enc"

Reply:
Oh. Well, then I guess the comment had the desired effect :-)

Reply:
> Oh. Well, then I guess the comment had the desired effect :-)

Just to be clear, this is OK, right? I did consider using sync's crypto here, but figured that added alot of complication for zero value and that having everything in the local DB use the same schema made more sense.

Reply:
Yes, seems fine; it may be worth a brief additional explanation about why, along the lines of what you say here. IIUC it's because:

* The mirror needs to have a fully-formed sync BSO payload, which includes the plaintext cc number.
* But we don't want plaintext cc numbers on disk, so we have to encrypt the whole payload for local storage.
* And the JWE encryption scheme is how we encrypt data for local storage, so that's what we use here.

Does that sound right?